client/src/linkManager.js

```diff
@@ -75,6 +88,23 @@ export class linkManager extends LuigiClientBase {
     this.navigate(path, 0, true, modalSettings || {});
   }
 
+  /**
+   * Opens a view in a splitted view. You can specify the split view's title and size. If you don't specify the title, it is the node label. If there is no node label, the title remains empty.  The default size of the split view is `l`, which means 80%. You can also use `m` (60%) and `s` (40%) to set the split view size. Optionally, use it in combination with any of the navigation functions.
+   * @memberof linkManager
+   * @param {string} path navigation path
+   * @param {Object} splitViewSettings opens a view in a split view. Use these settings to configure the split view's behaviour
+   * @param {string} splitViewSettings.title split view title. By default, it is the node label. If there is no label, it is left empty
+   * @param {number} [splitViewSettings.size=40] size of the split view in percent
+   * @returns {Object} instance of SplitView
+   * @see {@link splitView} for further documentation about the returned instance
+   * @example
+   * const splitViewHandle = LuigiClient.linkManager().openAsSplitView('projects/pr1/logs', {title: 'Logs', size: 40});
+   */
+  openAsSplitView(path, splitViewSettings) {
```

Comment:
I would vote to default `splitViewSettings` to `{}` so that we avoid any future runtime error for our customers in case of creating the split view handle with `undefined` splitViewSettings, and then we can remove the `or` in the next line ;)

Reply:
done